components/opc-ua/src/Microsoft.Azure.IIoT.OpcUa.Publisher/src/Services/PublisherJobService.cs

```diff
@@ -49,15 +49,15 @@ public sealed class PublisherJobService : IPublishServices<string> {
         });
 
         /// <summary>
-        /// Read default max outgress message buffer size from environment
+        /// Read default max egress message queue size from environment
         /// </summary>
-        internal Lazy<int> DefaultMaxOutgressMessages => new Lazy<int>(() => {
+        internal Lazy<int> DefaultMaxEgressMessageQueue => new Lazy<int>(() => {
             var env = Environment.GetEnvironmentVariable("PCS_DEFAULT_PUBLISH_MAX_OUTGRESS_MESSAGES");
```

Comment:
>"PCS_DEFAULT_PUBLISH_MAX_OUTGRESS_MESSAGES" [](start = 57, length = 43)

If was as bugfix implemented (quick and not so clean) when we reqork we should add the environment variable name to PcsVariables constant class.
Why didn't you also changed the environment variable? (it is not yet released)

Reply:
Done